src/styles/app.css

```diff
@@ -0,0 +1,12 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.nasa-logo {
+  margin: auto;
```

Comment:
The margin property does not work with the auto keyword for vertical directions, what youre effectively saying here is
```
margin-top: auto;
margin-right: auto;
margin-bottom: auto;
margin-right: auto;
```
This should be changed for 
```
margin: 0 auto;
```